mzLib/FlashLFQ/ChromatographicPeak.cs

```diff
@@ -16,20 +16,23 @@ public class ChromatographicPeak
         public int ScanCount => IsotopicEnvelopes.Count;
         public double SplitRT;
         public readonly bool IsMbrPeak;
```

Comment:
We are really bulking up ChromatographicPeak. Is there anything to the idea of grouping a bunch of stuff into another class ChromatographicPeakMetrics? Just asking.

Reply:
Honestly that would be a better way to do that. Create something like and FdrInfo type object. I hope to implement that change in a future update